File: pset1/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int height = get_int("Height:\n");

    while (true)
    {
        if (height >=1 && height <= 8)
        {
            switch(height)
            {
                case 1:
                    printf("#  #\n");
                    break;
                case 2:
                    printf(" #  #\n##  ##\n");
                    break;
                case 3:
                    printf("  #  #  \n ##  ## \n###  ###\n");
                    break;
                case 4:
                    printf("   #  #   \n  ##  ##  \n ###  ### \n####  ####\n");
                    break;
                case 5:
                    printf("    #  #    \n   ##  ##   \n  ###  ###  \n ####  #### \n#####  #####\n");
                    break;
                case 6:
                    printf("     #  #     \n    ##  ##    \n   ###  ###   \n  ####  ####  \n #####  ##### \n######  ######\n");
                    break;
                case 7:
                    printf("      #  #      \n     ##  ##     \n    ###  ###    \n   ####  ####   \n  #####  #####  \n ######  ###### \n#######  #######\n");
                    break;
                case 8:
                    printf("       #  #\n      ##  ##\n     ###  ###\n    ####  ####\n   #####  #####\n  ######  ######\n #######  #######\n########  ########\n");
                    break;
            }
            break;
        }
        else
        {
            height = get_int("Height:\n");
        }
    }
}

File: pset6/mario/less/mario.py
# Imports get_int function from cs50 library #
from cs50 import get_int

# Asks user for height input #
h = get_int("Height: ")

# Checks if input is in the valid range #
while (h < 1) or (h > 8):
    h = get_int("Height: ")
    if (h > 1) and (h < 8):
        break

else:
    # Loop creates half-pyramid according to height input #
    for x in range(h):
        for y in range(h - (x + 1)):
            print(" ", end="")
        for z in range(x + 1):
            print("#", end="")
        print()